Public/Sdk/Public/Managed/helpers.dsc

```diff
@@ -132,4 +136,43 @@ namespace Helpers {
 
         return results.toArray();
     };
+
+    /** TODO: Unofortunately the System.Interactive.Async package exposes IAsyncEnumerable in its own System.Generic.Collection namespace,
+    /*        colliding with .NETCore 3.0 implementation of the same name. For the time being (hopefully once the package maintainers updated their deps),
+    /*        we explicitly create an aliased reference for the code that depends on this.
+    **/
+
+    @@public
+    export function patchReferencesForSystemInteractiveAsync(references: Shared.Reference[]) : Csc.Arguments {
+        const needsSystemInteractiveAsync = references.some(ref =>
+            Shared.isManagedPackage(ref) && !Shared.isAssembly(ref) && ref.name.contains("System.Interactive.Async"));

```

Comment:
>!Shared.isAssembly(r [](start = 44, length = 20)

Curious why you needed this check? the first one should be exclusive...

Reply:
My best guess is we have precedence / ternary evaluation issues in DScript - as it did not work.